spec/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "jasmine": true,
+        "es6": false
```

Comment:
Are you suggesting that's how it _should_ work, or how it currently does? If the latter, I don't think so, this local config would extend, and overwrite, the higher level config